docs/3.x.x/guides/deployment.md

```diff
@@ -75,3 +83,317 @@ We highly recommend to use [pm2](https://github.com/Unitech/pm2/) to manage your
+
+```bash
+heroku config:set DATABASE_USERNAME=paulbocuse
+heroku config:set DATABASE_PASSWORD=mySecretPassword
+heroku config:set DATABASE_HOST="stapi-mongo-heroku-shard-00-00-fty6c.mongodb.net:27017,strapi-mongo-heroku-shard-00-01-fty6c.mongodb.net:27017,strapi-mongo-heroku-shard-00-02-fty6c.mongodb.net:27017/test?ssl=true&replicaSet=strapi-mongo-heroku-shard-0&authSource=admin&retryWrites=true"
+heroku config:set DATABASE_PORT=
+heroku config:set DATABASE_NAME=strapi-mongo-heroku
+```
+**Note:** Please replace these above values with the your actual values.
+
+##### 2. Update your database config file
+
+Replace the contents of `database.json` with the following:
+
+`Path: ./config/environments/production/database.json`.
+
+```json
+{
+  "defaultConnection": "default",
+  "connections": {
+    "default": {
+      "connector": "strapi-hook-mongoose",
+      "settings": {
+        "client": "mongo",
+        "host": "${process.env.DATABASE_HOST}",
+        "port": "${process.env.DATABASE_PORT}",
+        "database": "${process.env.DATABASE_NAME}",
+        "username": "${process.env.DATABASE_USERNAME}",
+        "password": "${process.env.DATABASE_PASSWORD}"
+      },
+      "options": {
+        "ssl": true
+      }
+    }
+  }
+}
+```
+
+::::
+
+### 6. Commit your changes
+
+`Path: ./my-project/`
+
+```bash
+git commit -am "Update database config"
+```
+
+### 7. Deploy
+
+`Path: ./my-project/`
+
+```bash
+git push heroku master
+```
+
+The deployment may take a few minutes. At the end, logs will display the url of your project (e.g. `https://mighty-taiga-80884.herokuapp.com`). You can also open your project using the command line:
+
+`Path: ./my-project/`
+
+```bash
+heroku open
+```
+
+If you see the Strapi Welcome page, you have correctly set-up, configured and deployed your Strapi project on Heroku. You will now need to set-up your `admin user` as the production database is brand-new (and empty).
+
+You can now continue with the [Tutorial - Creating an Admin User](/3.x.x/getting-started/quick-start-tutorial.html#_3-create-an-admin-user), if you have any questions on how to proceed.
+
+::: warning NOTE
+For security reasons, the Content Type Builder plugin is disabled in production. To update content structure, please make your changes locally and deploy again.
+:::
+
+---
+
+### Project updates
+
+You can continue developing your application with Strapi and Heroku by making changes on your computer and then using Git to commit and push those changes. After Heroku applies your changes, you can open your app.
+
+`Path: ./my-project/`
+
+```bash
+git add .
+git commit -am "Changes to my-project noted"
+git push heroku master
+heroku open
+```
+
+## AWS
+
+TBD.
+
+## Digital Ocean
```

Comment:
Vultr is another popular option, just some food for thought. Might be worth speaking to these companies to see if you can get a kickback with some discount links for users. :wink:

Reply:
We will add DO, Linode, etc doc examples also! its planned.  We're not stopping here.